docs-source/cloudbank/content/provision/install-free-trial.md

```diff
@@ -148,3 +148,72 @@ You can also use this approach in any regular commercial Oracle Cloud Tenancy -
    * The Spring Operations Center main dashboard will appear similar to this:
 
      ![Spring Operations Center](../images/install-free-tier-5.png " ")
+
+1. DB Access & setup
+
+   * Connect to your instance (using SSH), find the database pod with the following command: `kubectl get pods --all-namespaces`.
+
+      In the `oracle-database-operator-systemthen the address` namespace, look for a pods like `free-XXXX`.
+
+   * Access to DB instance with the following command: `kubectl exec -it  -n oracle-database-operator-system free-XXXX -- /bin/sh`
+
+   * Login as sysdba to create the user `account` that will be used in Module 3:  
+
+      ```sql
+      sqlplus / as sysdba
+      ALTER SESSION SET CONTAINER = freepdb1;    
+      CREATE TABLESPACE users DATAFILE '/opt/oracle/oradata/FREE/FREEPDB1/users.dbf'  SIZE 100M AUTOEXTEND ON NEXT 10M MAXSIZE 500M;
+      CREATE USER account IDENTIFIED BY Welcome1234## DEFAULT TABLESPACE users TEMPORARY TABLESPACE temp;
+      ```
+
+      This step replaces the steps from 1 to 4 in the task **Account Microservices/Prepare Database Objects**.
+
+      In Step 5 of **Prepare Database Objects**, skip the command: `create user account identified by "Welcome1234##";` and execute all the rest, since the user account has been already created.
```

Comment:
I'm not sure I follow, I can't find a Step 5?

Reply:
sure. done.